Update nws.markdown to match current behavior (#32272)
home-assistant/home-assistant.github.io: source/_integrations/nws.markdown

Providing a METAR station code is optional, and if not supplied, the closest station to the latitude and longitude will be chosen. A list of nearby stations is printed to the log with level `DEBUG` if no station is supplied. Stations can also be found on the [NOAA website](https://www.cnrfc.noaa.gov/metar.php). Codes with only three characters, for example, `ADW` should be prefixed with the letter K, `KADW`.

> One weather entity is created to provide forecasts for each entry in the configuration. The time supplied for each forecast is the start time for the forecast. Hourly forecast data can be obtained using the `weather.get_forecasts` service. Sensors are also created as disabled entities after configuration and can be enabled by the user.

Reviewer: ```suggestion
One weather entity is created for each entry in the configuration with hourly and day/night forecasts, which can be obtained through the `weather.get_forecasts` service. The time supplied for each forecast is the start time for the forecast. Sensors are also created as disabled entities after configuration and can be enabled by the user.
```

Author: I can commit that if you especially like it, but "with" threw me off briefly, so it might make sense to replace the word "with" using the word "providing".   I'm not sure that comma is grammatically correct, either, so maybe it should be broken into more sentences like this?:
> One weather entity is created for each entry in the configuration.  This entity provides hourly and day/night forecasts which can be viewed in the UI or accessed using the `weather.get_forecasts` service.

Even if that's better, I'm not sure if I should say "the UI" or "lovelace" though, so let me know what you think.

> One weather entity is created to provide forecasts for each entry in the configuration. The time supplied for each forecast is the start time for the forecast. Hourly forecast data can be obtained using the `weather.get_forecasts` service. Sensors are also created as disabled entities after configuration and can be enabled by the user.

Reviewer: What about this:

```suggestion
One weather entity is created for each entry in the configuration. Hourly and day/night forecasts are provided through the `weather.get_forecasts` service. The time supplied for each forecast is the start time for the forecast. Sensors are also created as disabled entities after configuration and can be enabled by the user.

```

Author: Technically sounds great, and I'm happy to commit it if you don't want to mention UI visibility.  The reason I mentioned UI visibility is because it might not be implicitly obvious to new users that the default dashboard and entity display already use this service, and that lack of knowledge might deter them from choosing this integration over others.

Reviewer: I actually dont know what you mean by this.  This integration to my knowledge is no different than other weather integrations.

Author: Other documentation I looked at was less thorough, and it just seemed to me like reading about the service could make it sound like users have to do something special when all they want is forecasts on their dashboard.  It's probably me overthinking this.

Reviewer: I'm not sure this belongs in the NWS documentation, since it applies to _all_ integrations with weather entities.  I would look to see if you think the description in the weather integration itself needs to be clarified.

> One weather entity is created for each entry in the configuration for day and night forecasts. The time supplied for each forecast is the start time for the forecast. Hourly forecast data can be obtained using the `weather.get_forecasts` service. Sensors are also created as disabled entities after configuration and can be enabled by the user.

Reviewer: The way you have worded this makes it sound like the entity is created for day/night forecast and the hourly forecast needs to be obtained via `get_forecasts`.  In reality, one entity is for both forecasts and `get_forecasts` can be used for either type.

Author: Should I change "day and night" to "day/night" or do you have another suggestion on how to clarify this?  Perhaps I need to read the latest change info and day/night isn't even accurate anymore, but that's the one entity I got when I installed in October...

Author: That verbiage was original verbiage that I didn't touch.  After reviewing the release notes, I have modified that language and added a commit.  I am marking this concern as resolved accordingly.

Details about the API are available in the [NWS API documentation](https://www.weather.gov/documentation/services-web-api). The [pynws](https://github.com/MatthewFlamm/pynws) library is used to retrieve data.

> Details about the `weather.get_forecasts` service are available in the [`weather` documentation](https://www.home-assistant.io/integrations/weather/).

Reviewer 2: Late review comment, there is a full link here, which shouldn't be used.

Please address this in a new PR to remove the `https://www.home-assistant.io` part from the target URL.

../Frenck

Reviewer 3: Thank you for the catch! https://github.com/home-assistant/home-assistant.io/pull/32348
